akka-docs/src/main/paradox/scala/common/other-modules.md

```diff
@@ -0,0 +1,36 @@
+# Other Akka modules
+
+* [Akka HTTP](http://doc.akka.io/docs/akka-http/current/scala.html)
+* [Alpakka](http://developer.lightbend.com/docs/alpakka/current/)
+* [Akka Streams Kafka](http://doc.akka.io/docs/akka-stream-kafka/current/home.html)
+* [Cassandra Plugins for Akka Persistence](https://github.com/akka/akka-persistence-cassandra)
+* [Akka Management](http://developer.lightbend.com/docs/akka-management/current/)
+
+## Community Projects
+
+Akka has a vibrant and passionate user community, the members of which have created many independent projects using Akka as well as extensions to it.
+
+ou can find Akka and Akka Http extensions, Akka Streams connectors and Akka Persistence plugins by exploring corresponding keywords in [Scaladex](https://index.scala-lang.org/):
```

Comment:
I wonder where should we draw the line between akka.io website and Akka documentation which is this project. This would get versioned and also will get into ZIPs PDFs and EPUBs (once paradox has that support). I think this content is more suited for the website.